eslzArm/managementGroupTemplates/policyAssignments/DINE-SQLThreatPolicyAssignment.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "topLevelManagementGroupPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Provide the ESLZ company prefix to the intermediate root management group containing the policy definitions."
+            }
+        },
+        "enforcementMode": {
+            "type": "string",
+            "allowedValues": [
+                "Default",
+                "DoNotEnforce"
+            ],
+            "defaultValue": "Default"
+        },
+        "nonComplianceMessagePlaceholder": {
+            "type": "string",
+            "defaultValue": "{enforcementMode}"
+        }
+    },
+    "variables": {
+        "policyDefinitions": {
+            "deploySqlThreat": "/providers/Microsoft.Authorization/policyDefinitions/36d49e87-48c4-4f2e-beed-ba4ed02b71f5"
+        },
+        "policyAssignmentNames": {
+            "deploySqlThreat": "Deploy-SQL-Threat",
+            "description": "This policy ensures that Threat Detection is enabled on SQL Servers.",
+            "displayName": "Deploy Threat Detection on SQL servers"
+        },
+        "nonComplianceMessage": {
+            "message": "Threat Detection {enforcementMode} be deployed on SQL servers.",
+            "Default": "must",
+            "DoNotEnforce": "should"
+        },
+        "rbacOwner": "8e3af657-a8ff-443c-a75c-2fe8c4bcb635",
```

Comment:
Can you refine this to not be owner please and use what the policy suggests of `SQL Security Manager` (`056cd41c-7e88-42e1-933e-88ba6a50c9c3`)


> https://www.azadvertizer.net/azpolicyadvertizer/36d49e87-48c4-4f2e-beed-ba4ed02b71f5.html